packages/svelte-cloudinary/tests/CldOgImage/CldOgImage.test.ts

```diff
@@ -0,0 +1,66 @@
+import { render, cleanup } from '@testing-library/svelte';
+import { afterEach, describe, expect, it, vi } from 'vitest';
+import { CldOgImage } from '$src/index';
+
+describe('CldImage', () => {
+	afterEach(() => {
+		cleanup();
+	});
+
+    it('should generate the correct URL for Cloudinary Open Graph image in the og:image meta tag', () => {
+        render(CldOgImage, {
+          props: {
+            src: 'og_image',
+
+            config: { cloud: { cloudName: 'testing-og' } },
+          },
+        });
+    
+        const metaTag = document.querySelector('meta[property="og:image"]');
+        
+        if (metaTag instanceof HTMLMetaElement) {
+          expect(metaTag.content).toContain('https://res.cloudinary.com');
+          expect(metaTag.content).toContain('og_image');
+        } else {
+          throw new Error('og:image meta tag not found');
+        }
```

Comment:
Could you change all these tests to type the metaTag, and use the expect instance of checker - something like this:

```js
const metaTag = document.querySelector<HTMLMetaElement>('meta[property="og:image"]');

expect(metaTag).toBeInstanceOf(HTMLMetaElement)
expect(metaTag?.content).toContain('https://res.cloudinary.com');
expect(metaTag?.content).toContain('og_image');
```